src/cli/index.js

```diff
@@ -23,6 +23,7 @@ import {getRcConfigForCwd, getRcArgs} from '../rc.js';
 import {spawnp, forkp} from '../util/child.js';
 import {version} from '../util/yarn-version.js';
 import handleSignals from '../util/signal-handler.js';
+import {boolify, boolifyWithDefault} from '../util/conversion.js';
```

Comment:
I'm not sure if `conversion` is a great module name. How about `user-input` or something like that? Or maybe `sanitization`?

Reply:
I don't really like any of those names, but can't come up with anything better.

I did use `conversion` because the functions convert things to boolean and maybe there might be other conversion functions some day that could also be placed in this module. `convert` would be the same but shorter.

If there is no good name and creating a new module for just these two functions is too much I could also add the functions to `util/misc.js`, but I wanted to avoid that.